src/Microsoft.DotNet.Build.CloudTestTasks/UploadToAzure.cs

```diff
@@ -91,31 +91,51 @@ public override bool Execute()
             }
 
             // first check what blobs are present
-            string checkListUrl = $"{AzureHelper.GetContainerRestUrl(AccountName, ContainerName)}?restype=container&comp=list"; 
+            string checkListUrl = $"{AzureHelper.GetContainerRestUrl(AccountName, ContainerName)}?restype=container&comp=list&maxresults=3000";
```

Comment:
What does `maxresults=3000` end up doing? Is there a default (lower, higher)?

Reply:
the default is actually based off the size of the response, so is a function of the length of the blobs' relative paths.  For your test container, the first response was around 4000 blobs.  This change is technically unnecessary but just makes the numbers for a guy stepping through with a debugger cleaner.